### ISSUE TITLE
WARNING [Dummy-488764] [version.py:36] - Check update version error: Invalid URL 'true': No scheme supplied. Perhaps you meant https://true?.

### ISSUE THREAD

**Issue by nowcoder-zhaozhen** [langgenius/dify#11118]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.10.1

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

I used AWS's claude model, and the API error log is as follows：WARNING [Dummy-488764] [version.py:36] - Check update version error: Invalid URL 'true': No scheme supplied. Perhaps you meant https://true?.

### ✔️ Expected Behavior

dify supports AWS's Gemini without any issues, but supports Claude exceptions

### ❌ Actual Behavior

dify supports AWS's Gemini without any issues, but supports Claude exceptions

**Comment by nowcoder-zhaozhen** [comment]
Does dify support AWS' Claude3.5 model？

**Comment by nowcoder-zhaozhen** [comment]
CHECK_UPDATE_URL: ${CHECK_UPDATE_URL:-https://updates.dify.ai}

**Comment by crazywoola** [comment]
Yes, please upgrade to latest version. We no longer fix bugs for 0.10.x
